sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -639,12 +639,14 @@ case class DescribeTableCommand(
  *    select * from (from a select * select *)
  * 7. Common table expressions (CTEs)
  */
-case class DescribeQueryCommand(query: LogicalPlan)
+case class DescribeQueryCommand(queryText: String, plan: LogicalPlan)
   extends DescribeCommandBase {
 
+  override def simpleString(maxFields: Int): String = s"$nodeName $queryText".trim
```

Comment:
So, we don't need `redact` here because this doesn't have CREATE statement. Am I right?

Reply:
@dongjoon-hyun uh... i am not sure. We are displaying the query that it input by the user.. so should be ok ?

Reply:
In general, it's not okay. In case of `CREATE TABLE ... USING JDBC` statement, we need to hide options like `id/password`.

Reply:
@dongjoon-hyun Got it.. yeah.. here we only take in queries.

Reply:
if users run EXPLAIN DESC QUERY CREATE TABLE ... USING JDBC, they already see these credentials?

Reply:
@cloud-fan they should see an error ? We only accept limited statements (select,cte) in our grammar ?

Reply:
@cloud-fan DESC QUERY CREATE.. should be rejected by our parser , right ?

Reply:
@cloud-fan Just got to computer :-). Here is the error we see : 
```
spark-sql> explain desc query create table q1(c1 int);
Error in query: 
mismatched input 'table' expecting {<EOF>, '.'}(line 1, pos 26)

== SQL ==
explain desc query create table q1(c1 int)
--------------------------^^^

```